stock_split_picking/wizards/stock_split_picking.py

```diff
@@ -43,6 +52,23 @@ def _apply_move(self):
                 new_pickings += picking._split_off_moves(move)
         return self._picking_action(new_pickings)
 
+    def _apply_available_line(self):
+        """Create different pickings for available and not available move line"""
+        for picking in self.mapped("picking_ids"):
+            if picking.picking_type_code == "outgoing":
+                moves = picking.mapped("move_ids")
+                moves_available = moves.filtered(lambda move: move.state == "confirmed")
```

Comment:
Wrong filter, available move are in the state "assigned" not "confirmed" 
```suggestion
                moves_available = moves.filtered(lambda move: move.state == "assigned")
```